domains/transfer/Transfer.API/API.IntegrationTests/Claiming/Api/Controllers/ClaimControllerTest.cs

```diff
@@ -0,0 +1,132 @@
+    {
+        this.factory = factory;
+    }
+
+    [Fact]
+    public async Task StopProcess_WhenClaimProcessExists_ReturnsNoContent()
+    {
+        var subject = Guid.NewGuid();
+        await factory.SeedClaims(new List<ClaimSubject>()
+        {
+            new(subject)
+        });
+        var client = factory.CreateAuthenticatedClient(sub: subject.ToString());
+
+        var result = await client.DeleteAsync("api/claim-automation/stop");
+        result.StatusCode.Should().Be(HttpStatusCode.NoContent);
+    }
+
+    [Fact]
+    public async Task StopProcess_WhenNoClaimProcessExists_ReturnsNotFound()
+    {
+        var subject = Guid.NewGuid();
+
+        var client = factory.CreateAuthenticatedClient(sub: subject.ToString());
+
+        var result = await client.DeleteAsync("api/claim-automation/stop");
+        result.StatusCode.Should().Be(HttpStatusCode.NotFound);
+    }
+
+    [Fact]
+    public async Task StartProcess_WhenNoClaimProcessHasStarted_ReturnsCreatedAt()
+    {
+        var subject = Guid.NewGuid();
+        var client = factory.CreateAuthenticatedClient(sub: subject.ToString());
+
+        var result = await client.PostAsync("api/claim-automation/start", null);
+        result.StatusCode.Should().Be(HttpStatusCode.Created);
+    }
+
+    [Fact]
+    public async Task StartProcess_WhenClaimProcessAlreadyStarted_ReturnsOk()
+    {
+        var subject = Guid.NewGuid();
+
+        await factory.SeedClaims(new List<ClaimSubject>()
+        {
+            new(subject)
+        });
+
+        var client = factory.CreateAuthenticatedClient(sub: subject.ToString());
+
+        var result = await client.PostAsync("api/claim-automation/start", null);
+        result.StatusCode.Should().Be(HttpStatusCode.OK);
+    }
+
+    [Fact]
+    public async Task GetClaimSubjectHistory_WhenHistoryEntryHasDifferentSubjectId_ReturnsNotFound()
+    {
+        const string actor = "Peter Producent";
+        var client1 = factory.CreateAuthenticatedClient(sub: Guid.NewGuid().ToString(), actor: actor);
+        await client1.PostAsync("api/claim-automation/start", null);
+
+        var client = factory.CreateAuthenticatedClient(sub: Guid.NewGuid().ToString());
+        var result = await client.GetAsync("api/claim-automation/history");
+
+        result.StatusCode.Should().Be(HttpStatusCode.NotFound);
+    }
+
+    [Fact]
+    public async Task GetClaimSubjectHistory_WhenHistoryExists_ReturnsOK()
+    {
+        const string actor = "Peter Producent";
+        var client = factory.CreateAuthenticatedClient(sub: Guid.NewGuid().ToString(), actor: actor);
+        await client.PostAsync("api/claim-automation/start", null);
+        var result = await client
+            .RepeatedlyGetUntil<ClaimSubjectHistoryEntriesDto>(
+                "api/claim-automation/history",
+                res => res.History.Any()
+            );
+
+        result.Should().NotBeNull();
+        result.History[0].ActorName.Should().Be(actor);
```

Comment:
It would make sense to assert on the entire resonse and not just a single property on the first history entry. You can use verfiy.xunit to make this a snapshot test

Reply:
Now uses verify